lib/logstash/inputs/stomp.rb

```diff
@@ -37,6 +37,12 @@ class LogStash::Inputs::Stomp < LogStash::Inputs::Base
   #Auto reconnect interval in seconds
   config :reconnect_interval, :validate => :number, :default => 30
 
+  # Enable TLS/SSL connection?
+  config :ssl, :validate => :boolean, :default => false
+
+  # Validate TLS/SSL certificate?
+  config :verify_ssl, :validate => :boolean, :default => false
```

Comment:
Two comments:

* Do we need this setting? Disabling SSL verification means you are turning off the security provided by SSL, and why are we using SSL in this case if we don't want security?

* This should default to true, not false.